finalitygadget/finalitygadget.go

```diff
@@ -336,8 +345,7 @@ func (fg *FinalityGadget) InsertBlock(block *types.Block) error {
 	if err != nil {
 		return err
 	}
-	// Set last finalized block in memory
```

Comment:
why we don't need to set it in memory anymore?

Reply:
we move [this](https://github.com/babylonlabs-io/finality-gadget/blob/fix/fg-activated-height/finalitygadget/finalitygadget.go#L442) into the function `handleBlock()`